packages/admin-panel-server/.env.example

```diff
@@ -3,8 +3,8 @@ DB_PASSWORD=
 DB_URL=
 DB_PORT=
 DB_USER=
+API_CLIENT_NAME=
+API_CLIENT_PASSWORD=
 ENTITY_API_URL=
-MEDITRAK_API_CLIENT_NAME=
-MEDITRAK_API_CLIENT_PASSWORD=
```

Comment:
Chose to rename these `API_CLIENT` env vars as they were a little unclear. Really these variables correspond the server's api client credentials (`admin-panel-server` in this instance).